index.md

```diff
@@ -1,11 +1,244 @@
+| Visit-detector images       |       15,972 |      2,805,017 |    x176 |          |
+| DiaObject Rows              |    1,089,818 |     41,301,558 |     x38 |          |
+| DiaSource Rows              |    3,086,404 |    162,448,407 |     x54 |          |
+| SSource                     |        5,988 |    653,005,444 | x109052 | (cg DP0.3 10-year)|
+| ForcedSourceOnDiaObject     |  196,911,566 | 16,817,191,952 |     x86 |          |
+
+One of the key observations that has arisen _since_ the launch of DP1 is that the DP0.2 userbase (and our internal scale testing campaign that drew for it) were a very poor predictor of actual user behavior with real data (for example types, rates and complexity of searches).
+
+Even within the Data Management teams who were already certain DP0.2 had instilled a false sense of confidence in a number of areas, it was an unpleasant surprise how little DP0.2 had prepared us for DP1.
+As an example, real-world users uncovered more issues than scale testing with 10x more users.
+Real users do wrong things, things you didn't expect, and generally don't behave like unit tests - which software teams are well aware of, but with the level of flexibility provided by the RSP, user-introduced entropy escalates much faster.
+There's a massive difference between "what if the user presses the wrong button" and "what if the user writes bad code", as there are only so many buttons but an infinite ways to  get code wrong.
+
+As a result, we have a critical need for a DP0.2-sized (minimum ~300 square degrees) data preview with real users working on LSSTCam data as an intermediate step before jumping to full DR1 size.
+
+**Recommendation:** A substantial DP2 release is needed to give us user behavior data on a genuine DR1 precursor while we still have the protection of the "preview" label so that we can explain to the community we are still learning. It should not be ruled out that an addition "preview" release will be required before we can stand behind something with the DR1 label.
+
+### Potential hybrid insufficiency
+
+Shortly after DP1 release, DP0.2 data was unavailable to data.lsst.cloud users (due to an S3DF compounded outage) for 9 days (DP1 catalogs were unavailable 16 hours longer than plannedq). In this light, the decision by Data Services to host DP1 images on Google Cloud may seem a fortuitous avoidance of a black swan event.
+
+What should not be lost is the reason for the decision to place DP1 next to the services in the first place.
+In the run-up to DP1, we found total end-to-end performance from the object store at S3DF to GGP was surprisingly poor, as well as highly variable.
+Additionally we had dropouts to communications to the Qserv REST API.
+Because of the DP1 schedule we did not have time to properly investigate these, and concerned that slow performance would seem unreasonable (and worrying) to users for such a small volume release, Data Services decided to host the DP1 images on Google Cloud.
```

Comment:
```suggestion
Because of the DP1 schedule we did not have time to properly investigate these and, concerned that slow performance would seem unreasonable (and worrying) to users for such a small volume release, Data Services decided to host the DP1 images on Google Cloud.
```